### PR TITLE
Template update nf-core tools v2.13.1

### PR DESCRIPTION
**Pull request by Daniel-VM**
## PR checklist

- [X] This comment contains a description of changes (with reason).
- [X] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/bacass/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/bacass _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [X] Make sure your code lints (`nf-core lint`).
- [X] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [X] Usage Documentation in `docs/usage.md` is updated.
- [X] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [X] `README.md` is updated (including new tool citations and authors/contributors).

## PR description
Update and merge of nf-core template update v2.13.1 (#114 )

### CODE REVIEW

**Review by FriederikeHanssen — APPROVED** [review]
lgtm 🚀

File: main.nf [hunk 1]
```diff
@@ -13,55 +13,86 @@ nextflow.enable.dsl = 2
...
-    def String command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv -profile docker"
-    log.info logo + paramsHelp(command) + citation + NfcoreTemplate.dashedLine(params.monochrome_logs)
-    System.exit(0)
-}
-
-// Validate input parameters
-if (params.validate_params) {
-    validateParameters()
-}
+include { getGenomeAttribute      } from './subworkflows/local/utils_nfcore_bacass_pipeline'
```
Comment (jonasscheid) [inline main.nf:24]: Do you need this function?

File: subworkflows/local/utils_nfcore_bacass_pipeline/main.nf [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+def toolBibliographyText() {
+    // TODO nf-core: Optionally add bibliographic entries to this list.
+    // Can use ternary operators to dynamically construct based conditions, e.g. params["run_xyz"] ? "<li>Author (2023) Pub name, Journal, DOI</li>" : "",
+    // Uncomment function in methodsDescriptionText to render in MultiQC report
+    def reference_text = [
+            "<li>Andrews S, (2010) FastQC, URL: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/).</li>",
+            "<li>Ewels, P., Magnusson, M., Lundin, S., & Käller, M. (2016). MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinformatics , 32(19), 3047–3048. doi: /10.1093/bioinformatics/btw354</li>"
+        ].join(' ').trim()
+
```
Comment (jonasscheid) [inline subworkflows/local/utils_nfcore_bacass_pipeline/main.nf:221]: Update tool citations

File: subworkflows/local/utils_nfcore_bacass_pipeline/main.nf [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+    meta["nodoi_text"] = meta.manifest_map.doi ? "": "<li>If available, make sure to update the text to include the Zenodo DOI of version of the pipeline used. </li>"
+
+    // Tool references
+    meta["tool_citations"] = ""
+    meta["tool_bibliography"] = ""
+
+    // TODO nf-core: Only uncomment below if logic in toolCitationText/toolBibliographyText has been filled!
+    // meta["tool_citations"] = toolCitationText().replaceAll(", \\.", ".").replaceAll("\\. \\.", ".").replaceAll(", \\.", ".")
+    // meta["tool_bibliography"] = toolBibliographyText()
+
```
Comment (jonasscheid) [inline subworkflows/local/utils_nfcore_bacass_pipeline/main.nf:242]: uncomment here after update of tool citations

File: subworkflows/local/utils_nfcore_bacass_pipeline/main.nf [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+//
+def toolCitationText() {
+    // TODO nf-core: Optionally add in-text citation tools to this list.
+    // Can use ternary operators to dynamically construct based conditions, e.g. params["run_xyz"] ? "Tool (Foo et al. 2023)" : "",
+    // Uncomment function in methodsDescriptionText to render in MultiQC report
+    def citation_text = [
+            "Tools used in the workflow included:",
+            "FastQC (Andrews 2010),",
+            "MultiQC (Ewels et al. 2016)",
+            "."
```
Comment (jonasscheid) [inline subworkflows/local/utils_nfcore_bacass_pipeline/main.nf:207]: Update tool citations

**Review by jonasscheid — CHANGES_REQUESTED** [review]
Awesome 🚀 ! Just update the tool citations and you are good to go!

File: main.nf (hunk 1 above)
Comment (Daniel-VM) [reply main.nf:24]: In fact, no 😅. I am going to remove it.

**Review by jonasscheid — APPROVED** [review]
LGTM 🚀 Always good to give the tools their credit!

**Comment by Daniel-VM** [comment]
Thank you!!! Yes, I completely forgot about that crutial part of the new version of the nf-core template.

**Pull request merged**